wip/meetings.ics

```diff
@@ -51,4 +51,79 @@ SEQUENCE:0
 STATUS:CONFIRMED
 SUMMARY:2020/03/26 TTWG meeting
 END:VEVENT
+BEGIN:VEVENT
+DTSTART:20200402T150000Z
+DTEND:20200402T160000Z
+UID:20200402@ttwg.w3.org
+CREATED:20200317T230000Z
+DTSTAMP:20200317T230000Z
+LAST-MODIFIED:20200317T230000Z
+DESCRIPTION:2020/04/02 TTWG meeting, https://github.com/w3c/ttwg/issues/105
+URL;VALUE=URI:https://github.com/w3c/ttwg/issues/105
+LOCATION:https://lists.w3.org/Archives/Member/member-tt/2016Oct/0009.html
+CATEGORIES:online-call
+SEQUENCE:0
+STATUS:CONFIRMED
+SUMMARY:2020/04/02 TTWG meeting
```

Comment:
When I open this file in Outlook for Mac it only shows one meeting, not each of the meetings. Would it make sense to have a separate ics file for each individual meeting? This seems like quite a lot of manual effort, so maybe there's a way to automate it...